lib/src/device.dart

```diff
@@ -1,20 +1,15 @@
 import 'dart:async';
-import 'dart:ui';
-
 import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
 
-import 'extensions/vector2.dart';
-
-/// Some utilities that did not fit anywhere else.
+/// Provides methods for controlling the device (e.g. setting the screen to fullscreen).
 ///
-/// To use this class, access it via [Flame.util].
-class Util {
+/// To use, access it via [Flame.device]
+class Device {
   /// Sets the app to be fullscreen (no buttons bar os notifications on top).
```

Comment:
Not your change, but within the parenthesis it should be something like:
"no buttons, bar or notifications on top"